src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -135,7 +135,71 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * BkBasic produces headers with addresses without slashes.
+     *
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void addressesInHeadersAddedWithoutSlashes() throws IOException {
+        final Socket socket = BkBasicTest.createMockSocket();
+        final Request[] holder = new Request[1];
```

Comment:
@alexey-krylov I think it can be like this
```
final Request request;
....
         public Response act(final Request req) {
              request = req;
                  return new RsEmpty();
....
).accept(socket);
```